.github/workflows/ci.yml

```diff
@@ -83,6 +83,12 @@ jobs:
             --host ${{ matrix.host_target }}
           rustup default master
 
+      # We need a nightly Cargo to run tests that depend on unstable Cargo features.
```

Comment:
Hm, I was quite happy when >1 year ago I switched us away from using a nightly cargo...

At least, could you put this above the previous job, so that the master toolchain is definitely the default?

Reply:
> At least, could you put this above the previous job, so that the master toolchain is definitely the default?

Hmm, I think I can also just ask rustup-toolchain-install-master to install Cargo...

Reply:
> Hmm, I think I can also just ask rustup-toolchain-install-master to install Cargo...

Yes that would be better I think.